examples/dapperish.go

```diff
@@ -19,10 +18,10 @@ import (
 func client() {
 	reader := bufio.NewReader(os.Stdin)
 	for {
-		span := opentracing.StartTrace("getInput")
-		ctx := opentracing.BackgroundGoContextWithSpan(span)
+		ctx, span := opentracing.BackgroundContextWithSpan(
```

Comment:
this file gives a sense of how user code differs... mainly just that `TraceContext` is no longer there.